lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py

```diff
@@ -3,61 +3,149 @@
 
 Sample usage::
 
-    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --app-csv "edx-platform Apps Ownership.csv"
+    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --repo-csv "Individual Repo Ownership.csv" --app-csv "edx-platform Apps Ownership.csv" --dep-csv "edx-platform 3rd-party Ownership.csv"
 
-Sample CSV input::
+Or for more details::
 
-    Path,owner.squad
-    ./common/djangoapps/xblock_django,team-red
-    ./openedx/core/djangoapps/xblock,team-red
-    ./lms/djangoapps/badges,team-blue
+    python lms/djangoapps/monitoring/scripts/generate_code_owner_mappings.py --help
 
-Sample output::
-
-    # Copy results into appropriate config yml file.
-    CODE_OWNER_MAPPINGS:
-      team-blue:
-      - badges
-      team-red:
-      - xblock_django
-      - openedx.core.djangoapps.xblock
 
 """
-import os
-import re
 import csv
 import click
+import os
+import re
+
+# Maps edx Django apps to their containing repo
```

Comment:
Mention Github org?

Reply:
I made this more clear by using the github urls, rather than a github repo name that was in the spreadsheet.